frontend/src/app/components/modals/create-namespace-bucket-modal/create-namespace-bucket-modal.html

```diff
@@ -86,6 +86,32 @@ <h2 class="heading3 push-prev">Write Policy</h2>
                 <wizard-controls></wizard-controls>
             </div>
         </section>
+
+        <section class="column greedy">
+            <div class="column greedy pad">
+                <p class="push-next">
+                    Caching configuration controls how data is cached in local storage
+                </p>
+                <editor class="greedy" params="label: 'Cache TTL'">
+                    <input type="number" min="0" max="172800"
```

Comment:
The max value of the input field does not match the new value in the `onValidate` method and `bucket_api` ( which is 3600 after the last changes).

You can add a property on the component class named `maxCacheTTL = 3600` (in the property declaration section`  and use it both in `onValidate` and here, by replacing `mak=172800` with `ko.attr.max=maxCacheTTL` os this value will be controlled from a single place.  

Please note that this value does not need to be `ko.observable` just a plain value (it is supposed to be constant)

Reply:
Ok.